ibm/mas_devops/roles/ibm_catalogs/tasks/install/main.yml

```diff
@@ -21,34 +21,52 @@
 # 2.2 Offline catalog
 # If this fails then it means we can't perform an airgap install using this catalog
 - name: Load CASE bundle versions for the offline catalog
-  when: airgap_install
+  when:
+    - airgap_install
   include_vars:
     file: "{{ role_path }}/../../common_vars/casebundles/{{ mas_catalog_version }}.yml"
 
 # If we are updating the catalog in an airgap environment, we first have to
 # update the image digest configmaps, otherwise when the operators update
 # they will not be able to use digests for the new images.
 - name: "Update digest maps ahead of catalog update"
-  when: airgap_install
+  when:
+    - airgap_install
+    - not dev_mode_airgap
   include_tasks: tasks/install/update-digest-maps.yml
 
 - name: "Create IBM offline catalog"
-  when: airgap_install
+  when:
+    - airgap_install
```

Comment:
We should be able to use:

```
    - artifactory_username is not defined or artifactory_username == ""
    - artifactory_apikey is not defined or artifactory_apikey == ""
```

the same as we use this everywhere else in the collection today to detect "pre-release mode", removing the need for the `not dev_mode_airgap` condition.